features/facebookImport/src/views/import/import.jsx

```diff
@@ -20,7 +20,7 @@ const Import = () => {
         handleImportFile,
     } = useContext(ImporterContext);
     const importStatus = navigationState.importStatus;
-    const file = files[0];
+    const file = files?.[0];
```

Comment:
So `file` can be `null`, right?

Reply:
Yeah - but the code already accounts for that. What's new is that `files` can be `null` - I used that as a loading marker 😈

Reply:
We are keeping files because we think multiple files will be coming back at some point right? If not we could get rid of it at some point also

Reply:
My point is that there will be no provision for rendering in that case.
Also, using literals as markers for something else will come back to bite you. Still approved, anyway.

Reply:
:D Yeah, we're keeping the array for now since the whole code can deal with a list of files, and that's likely what we do next.

If it was a single file, what I did here would _not_ work. We used to use an empty array until all files are loaded, so we couldn't distinguish between _there are no files_ and _files are being loaded_. That's why I added `null` for the loading state. Of course @JJ is right - it's a bit of a hack - the fact that I felt like adding a comment (I hate comments! :D) says something.